docs/API_walkthrough.md

```diff
@@ -0,0 +1,644 @@
+# Quickstart
+
+This quickstart guide shows a real use case scenario of interaction with the API in an Apiculture context.
+
+## Prepare your environment
+
+The provided examples make use of the HTTPie command line tool (installation instructions: https://httpie.org/docs#installation)
+
+All requests are grouped in a Postman collection that can be found [here](https://github.com/stellio-hub/stellio-docs/blob/master/API_Quick_Start.postman_collection.json).
+For more details about how to import a Postman collection see https://learning.postman.com/docs/getting-started/importing-and-exporting-data/.
+
+Export the link to the JSON-LD context used in this use case in an environment variable for easier referencing in
+the requests:
+
+````shell
+export CONTEXT_LINK="<https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld>; rel=http://www.w3.org/ns/json-ld#context; type=application/ld+json"
+````
+
+## Starting the Stellio Context Broker
+The Stellio Contex Broker source code can be found here : https://github.com/stellio-hub/stellio-context-broker
+
+To Start a Stellio instance. You can use the provided Docker compose configuration:
+
+```shell
+http GET https://raw.githubusercontent.com/stellio-hub/stellio-context-broker/develop/docker-compose.yml
+```
+and then start the container with:
+```shell
+docker-compose -f docker-compose.yml up -d && docker-compose -f docker-compose.yml logs -f
```

Comment:
without the `.env` file, I don't think it will work.

Reply:
Ok.
We need to clone the project instead of getting only the docker compose file?, Or to get the .env file with the docker compose?

Reply:
I have made a proposition, tell me if it is ok.